src/components/v5/common/MembersList/MembersList.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import { MembersListProps } from './types';
+import CardWithBios from '~v5/shared/CardWithBios/CardWithBios';
+import EmptyContent from '../EmptyContent';
+import { useMembersList } from './hooks';
+import Link from '~v5/shared/Link';
+
+const displayName = 'v5.common.MembersList';
+
+const MembersList: FC<MembersListProps> = ({
+  title,
+  description,
+  list,
+  isLoading,
+  emptyTitle,
+  emptyDescription,
+  viewMoreUrl,
+}) => {
+  const { formatMessage } = useIntl();
+  const { handleClipboardCopy } = useMembersList();
+
+  const listLength = list.length;
+
+  return (
+    <div>
+      <div className="flex items-center mb-2">
+        <h3 className="heading-5 mr-3">{formatMessage(title)}</h3>
+        <span className="text-md text-blue-400">
+          {listLength} {formatMessage(title)}
+        </span>
+      </div>
+      <p className="mb-6">{formatMessage(description)}</p>
+      {!isLoading && (
+        <ul>
+          {listLength > 0 ? (
```

Comment:
`!!listLength`

Reply:
It was meant to be listLength > 8 and not 0, so I've fixed that